rust_dev_preview/iam/README.md

```diff
@@ -1,58 +1,72 @@
+  the task. For more information, see
+  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information,
+  see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
 ## Code examples
 
-- [Create a role](src/bin/create-role.rs) (CreateRole)
+### Single action
 
-## ⚠ Important
+* [Listing Roles](src/iam-service-lib.rs) (ListRoles)
+* [Listing Users](src/iam-service-lib.rs) (ListUsers)
+* [Listing Groups](src/iam-service-lib.rs) (ListGroups)
+* [Attaching Role Policies](src/iam-service-lib.rs) (AttachRolePolicy)
+* [Getting an Account Password Policy](src/iam-service-lib.rs) (GetAccountPasswordPolicy)
+* [Listing SAML Providers](src/iam-service-lib.rs) (ListSAMLProviders)
+* [Listing Role Policies](src/iam-service-lib.rs) (ListRolePolicies)
+* [Getting a Role](src/iam-service-lib.rs) (GetRole)
+* [Listing Policies](src/iam-service-lib.rs) (ListPolicies)
+* [Creating a Service Linked Role](src/iam-service-lib.rs) (CreateServiceLinkedRole)
+* [Creating a Role](src/iam-service-lib.rs) (CreateRole)
+* [Listing Attached Role Policies](src/iam-service-lib.rs) (ListAttachedRolePolicies)
+* [Getting a Policy](src/iam-service-lib.rs) (GetPolicy)
 
-- We recommend that you grant this code least privilege, 
-  or at most the minimum permissions required to perform the task.
-  For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions.
-  Some AWS services are available only in specific
-  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account.
+### Scenario
 
-## Running the code examples
+* [Getting started with IAM](src/bin/iam-getting-started.rs)
 
-### Prerequisites
+## Running the code
 
-You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+Run the scenario with the following command:
 
-### create-role.rs
+```
+cargo run --bin iam_getting_started
+```
 
-This example creates an IAM role in the Region.
+### Prerequisites
 
-`cargo run --bin create-role -- -a ACCOUNT-ID -b BUCKET -n NAME -p POLICY-FILE [-r REGION] [-v]`
+- You must have an AWS account, and have configured your default credentials and AWS Region as described in [https://github.com/awslabs/aws-sdk-rust](https://github.com/awslabs/aws-sdk-rust).
+- Rust and Cargo installed, as described [in the Rust documentation](https://doc.rust-lang.org/book/ch01-01-installation.html)
```

Comment:
Install Rust and Cargo as described [in the Rust documentation](https://doc.rust-lang.org/book/ch01-01-installation.html).

Reply:
fixed